styles.css

```diff
@@ -0,0 +1,222 @@
+@import url('https://fonts.googleapis.com/css2?family=Crete+Round&family=Domine:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap');
+
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 125vh;
+  background: #1c1a19;
+  background-size: cover;
+}
+
+.portfolio {
+  width: 100%;
+  height: 125vh;
+  left: 0;
+  top: 0;
+  margin: 0 auto;
+}
+
+.details {
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  width: 100%;
+  height: auto;
+  left: 0;
+  top: 0;
+  background: #3e3c3c;
+  flex-wrap: wrap;
+}
+
+.inside-name {
+  width: 92px;
+  height: 20px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 20px;
+  color: #fff;
+}
+
+.section-container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 10px 0;
+  gap: 12px;
+}
+
+.greeting {
+  width: 327px;
+  height: 124px;
+  font-family: 'Crete Round', serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 72px;
+  line-height: 82px;
+  color: #f55800;
+  margin: 24px 0 24px 0;
+}
+
+.introduction {
+  width: 566px;
+  height: 24px;
+  font-family: 'Inter', sans-serif;
+  font-weight: 500;
+  font-size: 32px;
+  line-height: 44px;
+  color: #fff;
+  margin: 24px 0 24px 0;
+}
+
+.paragraph {
+  width: 566px;
+  height: 123px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #fff;
+  margin: 24px 0 24px 0;
+}
+
+.socials-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  position: absolute;
+  margin: 270px 0 0 30px;
+  order: 0;
+}
+
+.socials-container a {
+  margin: 20px 0 20px 0;
+}
+
+@media all and (max-width: 375px) {
+  .details {
+    background: #3e3c3c;
+    width: 375px;
+    height: 70px;
+  }
+
+  body {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    height: auto;
+    background: #1c1a19;
+    background-size: cover;
+  }
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design
  - You have no background image as in the Figma design.
  - Your hamburger or images icon is not showing.

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2023-01-20 00-30-57](https://user-images.githubusercontent.com/92303891/213592243-fa2f3b5a-53c9-41d1-ae1c-57bc49977cd3.png)| ![Screenshot from 2023-01-20 00-26-41](https://user-images.githubusercontent.com/92303891/213591743-faccead1-ddbb-454d-bc94-a43d5b542e6a.png)|